multicluster/cmd/link.go

```diff
@@ -323,9 +325,32 @@ A full list of configurable values can be found at https://github.com/linkerd/li
 	cmd.Flags().StringVarP(&opts.selector, "selector", "l", opts.selector, "Selector (label query) to filter which services in the target cluster to mirror")
 	cmd.Flags().StringVar(&opts.gatewayAddresses, "gateway-addresses", opts.gatewayAddresses, "If specified overwrites gateway addresses when gateway service is not type LoadBalancer (comma separated list)")
 
+	pkgcmd.ConfigureNamespaceFlagCompletion(
+		cmd, []string{"namespace", "gateway-namespace"},
+		kubeconfigPath, impersonate, impersonateGroup, kubeContext)
+	configureClusterNameFlagCompletion(cmd)
```

Comment:
Since `linkerd mc link` creates a new link that is named with `--cluster-name`, it shouldn't do any completion. It should though for `linkerd mc unlink`